docker-compose.yml

```diff
@@ -34,15 +34,13 @@ services:
       context: .
       target: production
       dockerfile: apps/web/docker/Dockerfile
-    # This enables to have this service for testing production build
-    # It will not run when doing docker compose up
-    profiles: [building]
-    volumes:
-      - ./packages/env:/app/packages/env
-      - ./packages/core:/app/packages/core
-      - ./packages/jobs:/app/packages/jobs
-      - ./packages/web-ui:/app/packages/web-ui
-      - ./apps/web:/app/apps/web
```

Comment:
I find this helpful

Reply:
why? are u using the dockerfile for development?